client/src/views/TableDetail.vue

```diff
@@ -36,7 +32,7 @@ export default {
     }
   },
   methods: {
-    classNameByIndex(index) {
+    rowClassName(index) {
```

Comment:
Does this method get called anywhere? I expected that if the name changed, some invocation of it would change as well.

Reply:
It gets called on line 13

Reply:
Ah, thanks, completely missed it!